packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -96,17 +96,19 @@ export async function saveProjectIdToAppConfigAsync(
     case 'success':
       break;
     case 'warn': {
-      Log.log();
+      Log.warn();
       Log.warn('It looks like you are using a dynamic configuration!');
```

Comment:
```suggestion
      Log.warn(`Warning: Your project uses dynamic app configuration (${chalk.bold('app.config.js')}), and the EAS project ID can't automatically be added to it.`);
```

replace app.config.js above with the dynamic config file used (app.config.js or ts). this is included in the default `message` (https://github.com/expo/expo/blob/a1b2dcb7558eeb126a4c71322c7df5552923ad59/packages/%40expo/config/src/Config.ts#L407-L410) but it's probably easiest to just get the value again with [`getConfigFilePaths`](https://github.com/expo/expo/blob/a1b2dcb7558eeb126a4c71322c7df5552923ad59/packages/%40expo/config/src/Config.ts#L263-L278)